docs/data/components/select/select.mdx

```diff
@@ -110,6 +110,18 @@ The `Select.ItemIndicator` subcomponent renders an indicator inside an option to
 </Select.ItemIndicator>
 ```
 
+## Modality
+
+By default, the Select is non-modal, so elements outside it can be interacted with.
+To make it modal, use the `modal` prop.
+You may also add a backdrop to the modal Select by using the `Select.Backdrop` component, to indicate that the outside elements are inert.
+
+```jsx
+<Select.Root modal>
+  <Select.Backdrop />
+</Select.Root>
+```
```

Comment:
I guess if `alignItemToTrigger` is true, it should be `modal` by default. The reason is you can scroll the background but it doesn't reposition itself unlike with regular popup-aligned anchoring, plus scrolling is locked which seems like it should be modal-only feature

Reply:
I suppose we could even make it modal by default all the time (that's how native macOS 
 controls work). cc @vladmoroz